applications/machine_learning/src/modules/ei_data_forwarder_bt_nus.c

```diff
@@ -342,8 +367,7 @@ static bool handle_ble_peer_event(const struct ble_peer_event *event)
 	case PEER_STATE_DISCONNECTED:
 		k_work_cancel(&send_queued);
 	case PEER_STATE_DISCONNECTING:
-		conn_state &= ~CONN_SECURED;
-		conn_state &= ~CONN_INTERVAL_VALID;
+		conn_state = 0;
```

Comment:
this is done in a quite magic way. In other pleases you are using enum type, why not add '0' state enum and us it here?

Reply:
I think it is just flags clearing.

Reply:
That clears all of the flags. Should I change it to using enum values explicitly ?

Reply:
I misunderstood concept of bitfields.

Reply:
Discussed offline, added comment since this assignment could be misleading.